lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_generate_po_file_from_metadata.rb

```diff
@@ -0,0 +1,136 @@
+require_relative '../../helper/po_extended'
+require 'fastlane/action'
+require 'gettext/po'
+
+module Fastlane
+  module Actions
+    class AndroidGeneratePoFileFromMetadataAction < Action
+
+      # SPECIAL_KEYS are keys that need to be treated specially
+      SPECIAL_KEYS = %w[release_notes release_notes_previous].freeze
+      REQUIRED_KEYS_TO_COMMENT_HASH = {
+        full_description: 'Multi-paragraph text used to display in the Play Store. Limit to 4000 characters including spaces and commas!',
+        title: 'Title to be displayed in the Play Store. Limit to 30 characters including spaces and commas!',
+        short_description: 'Short description of the app to be displayed in the Play Store. Limit to 80 characters including spaces and commas!',
+        release_notes_short: nil,
+        release_notes: 'Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!',
+        release_notes_previous: nil
+      }.freeze
+
+      REQUIRED_KEYS = REQUIRED_KEYS_TO_COMMENT_HASH.keys.map(&:to_s).freeze
+
+      OPTIONAL_KEYS_TO_LABELS_HASH = {
+        # optional_key_label: 'Description for the first app store image',
+      }.freeze
+      KEYS_TO_COMMENT_HASH = REQUIRED_KEYS_TO_COMMENT_HASH.merge(OPTIONAL_KEYS_TO_LABELS_HASH).freeze
+
+
+      def self.required_keys
+        REQUIRED_KEYS
+      end
+
+      def self.run(params)
+        @metadata_directory = params[:metadata_directory]
+        @release_version = params[:release_version]
+        @other_sources = params[:other_sources]
+
+        prefix = 'play_store'
```

Comment:
> 💄 Nit: I'd include the ending `_` to this prefix. That way, if we decide later to remove this prefix we can just set it to `''`, or if we want to change it to `play_store-` instead of `play_store_` we easily could too. 🙃

it was on my Todo list, can fix immediatly.